Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html

```diff
@@ -0,0 +1,91 @@
+<html>
+
+<head>
+    <title>
+        Assignment6
+    </title>
+    <link rel="stylesheet" href="PreetiA6_1.css">
+</head>
+
+<body>
+    <h1>MI-9 Intern HTML Assessment</h1>
+    <hr>
+    <h3>Personal details</h3>
+    <form>
+        <div id="basicDetails">
+            <label for="fn">Firstname:</label>
+            <input type="text" id="fn" name="fname"><br><br>
+            <label for="ln">Lastname:</label>
+            <input type="text" id="ln" name="lname"><br><br>
+            <p>Gender:</p>
+            <input type="radio" id="m" name="male">
+            <label for="m">Male</label>
+            <input type="radio" id="f" name="female">
+            <label for="f">Female</label>
+            <input type="radio" id="o" name="others">
+            <label for="o">Others</label>
+            <p>Age:</p>
+            <input type="radio" id="less" name="age1">
+            <label for="less">less than 18</label>
+            <input type="radio" id="equal" name="age2" checked>

```

Comment:
no need of check here

Reply:
@Pawan1005 please elaborate which attribute I need to check?

Reply:
> check attribute

please elaborate which attribute I need to check?

Reply:
for checked